src/textual/widgets/_select.py

```diff
@@ -278,6 +278,44 @@ def __init__(
         self._value: SelectType | None = value
         self._options = options
 
+    @classmethod
+    def from_values(
+        cls,
+        opts: [SelectType],
+        *,
+        prompt: str = "Select",
+        allow_blank: bool = True,
+        value: SelectType | None = None,
+        name: str | None = None,
+        id: str | None = None,
+        classes: str | None = None,
+        disabled: bool = False,
+    ):
+        """Initialize the Select control with options specified by an iterator
+
+        Args:
+         options: Options to select from.
```

Comment:
I think users could benefit from a sentence indicating that we specify the values (like you said) and that the string representations are generated automatically by passing each option to the built-in `str`.

In other words, I want it to be absolutely clear for users that this works with options other than strings.

Reply:
I've fixed this in my latest set of commits (clarified that the iterator can be of any type).  Let me know if the changes are OK or if they need to be tweaked any more

Reply:
Your new docstring was much better, thanks.